pkg/adapter/v2/cloudevents_test.go

```diff
@@ -181,7 +193,10 @@ func TestNewCloudEventsClient_send(t *testing.T) {
 					t.Fatal(err)
 				}
 				validateSent(t, innerClient, tc.event.Type())
-				validateMetric(t, got.reporter, 1, tc.wantRetryCount)
+				if tc.eventCount == 0 {
+					tc.eventCount = 1
+				}
```

Comment:
this feels gross but i didn't want to have to add `eventCount: 1` to every test and couldn't think of how else to default it to 1

Reply:
I think we should just add `eventCount: 1` to each test where that is true, or change it to a "retry" boolean that means eventCount will be 1+1, or else it is just 1